### PR TITLE
Profile addition

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
Adds my profile to linktree.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/245"></a>

### CODE REVIEW

File: public/data/AbhijithGanesh.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "url":"https://github.com/AbhijithGanesh",
+            "icon":"github"
+        },
+        {
+            "name":"Conenct with me on LinkedIn",
+            "url":"https://linkedin.com/in/AbhijithGanesh14",
+            "icon":"linkedin"
+        },
+        {
+            "name":"Check my website out",
```
Comment (rohan-kulkarni-25) [inline public/data/AbhijithGanesh.json:17]: you can write " Check Out my website " That will look more proper ( just a suggestion )

File: src/Components/Links.js [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 import { Button } from 'primereact/button'
 
 function Links({ links }) {
   const colors = {
     youtube: '#FF0000',
     twitter: '#00ACEE',
     github: '#171515',
+    linkedin: '#0a66c2',
+    web: '#000000',
+    tiktok: '#010101',
```
Comment (rohan-kulkarni-25) [inline src/Components/Links.js:12]: I think than including all Socials a few will be proper which everyone is using

File: public/index.html [hunk 3]
```diff
@@ -5,39 +5,11 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
```
Comment (rohan-kulkarni-25) [inline public/index.html:7]: Please check Index.html file once it seems you deleted few lines

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

File: public/index.html [hunk 4]
```diff
@@ -5,39 +5,11 @@
...
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
```
Comment (eddiejaoude) [inline public/index.html:10]: We should probably update this rather than delete it

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.
You may also want to re-make this PR as it's outdated and the branch is deleted.

**Comment by eddiejaoude** [comment]
Looks like this is a duplicate, I will close https://github.com/EddieHubCommunity/LinkFree/blob/main/public/data/AbhijithGanesh.json

**Pull request closed without merging**